### PR TITLE
GP2 224 upcoming and past events on projects tab

### PR DESCRIPTION
**Pull request by markcsims**
- empty commit to create pr
- adds events upcoming and past to working groups
- adds test coverage for events
- adds missing coverage
- tests correct count is displayed
- starts adding events to projects page

### CODE REVIEW

File: packages/gp2-components/src/organisms/ProjectDetailHeader.tsx [hunk 1]
```diff
@@ -75,69 +77,82 @@ const ProjectDetailHeader: React.FC<ProjectDetailHeaderProps> = ({
...
-            </Paragraph>
-            <Link href={opportunitiesLink}>
-              <span css={css({ display: 'inline-flex' })}>
-                Read more <span css={infoIconStyles}>{externalLinkIcon}</span>
-              </span>
-            </Link>
+  upcomingTotal = 0,
+  pastTotal = 0,
+}) => {
+  const route = gp2Routing.projects({}).project({ projectId: id });
```
Comment (jhonny-m) [inline packages/gp2-components/src/organisms/ProjectDetailHeader.tsx:83]: we are only using it in one of the tablink, either we use it more or remove it 😄

File: packages/frontend-utils/src/events/index.ts [hunk 2]
```diff
@@ -52,14 +50,6 @@ export const getEventListOptions = <C extends Constraint = EventConstraint>(
...
-    ...(past
-      ? {
-          before: time,
-          sort: {
-            sortBy: 'endDate',
-            sortOrder: 'desc' as const,
-          },
-        }
-      : { after: time }),
+    ...(past ? { before: time } : { after: time }),
```
Comment (jhonny-m) [inline packages/frontend-utils/src/events/index.ts:53]: how does the ordering got changed?

File: packages/gp2-components/src/organisms/ProjectDetailHeader.tsx (hunk 1 above)
Comment (markcsims) [reply packages/gp2-components/src/organisms/ProjectDetailHeader.tsx:83]: We use it in 5 places.

File: packages/frontend-utils/src/events/index.ts (hunk 2 above)
Comment (markcsims) [reply packages/frontend-utils/src/events/index.ts:53]: 1. The sort is no longer used by crn, the events are sourced from Algolia and this property is no longer used.
2. The past events should be ordered by endData desc. GP2 wasn't doing this.

Comment (jhonny-m) [reply packages/frontend-utils/src/events/index.ts:53]: and we are gonna do it now?

Comment (jhonny-m) [reply packages/frontend-utils/src/events/index.ts:53]: I'm just trying to understand if there is work missing for events

**Review by jhonny-m — APPROVED** [review]

**Pull request merged**